third-party/CMakeLists.txt

```diff
@@ -75,8 +75,8 @@ endif()
 
 include(ExternalProject)
 
-set(LIBUV_URL https://github.com/libuv/libuv/archive/v1.8.0.tar.gz)
-set(LIBUV_SHA256 906e1a5c673c95cb261adeacdb7308a65b4a8f7c9c50d85f3021364951fa9cde)
+set(LIBUV_URL https://github.com/libuv/libuv/archive/v1.9.1.tar.gz)
+set(LIBUV_SHA256 e83953782c916d7822ef0b94e8115ce5756fab5300cca173f0de5f5b0e0ae92)
```

Comment:
I don't know why there's a difference, but I get: `a6ca9f0648973d1463f46b495ce546ddcbe7cce2f04b32e802a15539e46c57ad`

The extracted archive looks fine too.